### PR TITLE
added documentation of set_local_revision and working with branches vs vagrant

### PR DESCRIPTION
**Pull request by de-code**
so that I don't need to ask you again..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,17 @@ Note: if you with to use a private key not in `~/.ssh/id_rsa`, you can [customiz
 Note: if you wish to use a hypervisor other than `virtualbox`, you can use the `vagrant-mutate` plugin
 to rebuild the ubuntu/trusty64 box for your own hypervisor.  See the [vagrant and virtualbox documentation](docs/vagrant-and-virtualbox.md).
 
+#### Working with formula branches
+
+Formula projects cloned to the local _cloned-projects_ directory. Changes will be picked up from there and branches can be used as well.
```
Comment (giorgiosironi) [inline README.md]: `are cloned`?

File: README.md [hunk 2]
```diff
@@ -57,6 +57,17 @@ Note: if you with to use a private key not in `~/.ssh/id_rsa`, you can [customiz
...
+
+#### Working with project branches
+
+After starting (`vagrant up`) and login in to (`vagrant ssh`) the VM, run the following command to change the commit or branch:
+
+    $ set_local_revision $commitOrBranch
+
+Provisioning (`vagrant provision` or `sudo salt-call state.highstate`) will then checkout the particular commit or branch.
```
Comment (giorgiosironi) [inline README.md]: May be worth pointing out that this branch must be present on the origin, it can't be local.

File: README.md (hunk 1 above)
Comment (de-code) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (de-code) [reply README.md]: added _remote_

**Pull request merged**